docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+Both approaches are described below.
+
+.. todo::
+
+  We don't totally understand why GBD models *excess* risk -- while there is a possibility that
+  there is a good reason behind it, we haven't been able to find that reason, so we prefer
+  Approach 1 for now.
+
+If GBD does not report a mediation factor, it may still report one or more of the effects in the diagram.
+The rest can be filled in with external sources.
+
+.. note::
+  Be careful not to interpret GBD RRs as *direct* RRs -- unless there is a mediation factor,
+  the GBD RR will not be adjusted for any mediators.
+
+All our multiplicative math is based on this equation describing how RRs are applied:
+
+.. math::
+
+  RR_\text{r→o} = RR_\text{r→o,direct} * \prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}
+
+where :math:`M` is the set of mediators that mediate the relationship between risk :math:`r`
+and outcome :math:`o` and :math:`\Delta_\text{r→m}` is the (assumed linear) increase in :math:`m`
+caused by a one-unit increase in :math:`r`.
+
+.. note::
+  This ignores the concept of a TMREL below which the RR does not apply and risk
+  is flat.
+  This seems to be ignored throughout the GBD's descriptions of its mediation methods, although
+  some risks that are mediators in GBD, such as BMI, clearly have non-zero TMRELs.
+
+.. note::
+  :math:`\Delta_\text{r→m}` only makes sense if both the risk and the mediator are continuous values.
+  Since all of the mediation we have modeled so far has been between continuous
+  risk factors, we haven't yet explored how to make different assumptions (neither has GBD).
+
+Approach 1: consistent with RR application, inconsistent with GBD mediation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this approach, we work backwards from the result reported by GBD (the mediation factor)
+to an underlying data point we can use to create an internally-consistent set of causal effects.
+It is an "undo and redo" approach: we undo the calculations GBD did in additive space to infer
+the data they used as inputs, then do a completely different calculation on those data in multiplicative space.
+To do this, we have to know whether the mediation factor for the pathway in question was calculated
+with the adjusted RR method ("Comparing crude RR versus mediator-adjusted RR"
+in `the GBD risk factors methods appendix [supplementary appendix 1] <https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30752-2/fulltext#supplementaryMaterial>`_) or
+the delta method ("Estimating the mediation factor by pathway of the effect").
+
+.. todo::
+
+  Where should someone look to find this out?
+
+Adjusted RR method
+^^^^^^^^^^^^^^^^^^
+
+In this case, GBD's data were a pair of RRs for the risk-outcome pair,
+one adjusted for the mediator and one not.
+We presume that the unadjusted RR is (approximately) equal to the total RR
+GBD reports for the risk-outcome pair.
```

Comment:
Should we note that (presumably) both of these RRs are adjusted for other confounders besides the mediator?